core/src/main/resources/hudson/model/AbstractModelObject/error.jelly

```diff
@@ -24,11 +24,15 @@ THE SOFTWARE.
 
 <?jelly escape-by-default='true'?>
 <j:jelly xmlns:j="jelly:core" xmlns:st="jelly:stapler" xmlns:l="/lib/layout">
-  <st:statusCode value="400" /><!-- clearly indicate that this page is an error page --> 
+  <st:statusCode value="400" /><!-- clearly indicate that this page is an error page -->
   <st:setHeader name="X-Error" value="${message}"/>
   <l:layout title="Jenkins">
     <l:header />
-    <l:side-panel />
+    <l:side-panel>
+      <l:tasks>
+        <l:task href="configure" icon="icon-up" title="${%Back to Configure}"/>
```

Comment:
```suggestion
        <l:task href="configure" icon="icon-up" title="${%Back to user configuration}"/>
```
'Configure' itself ready a bit weird, no?

Reply:
Indeed, if we use "Back to user configuration", it may be better to let the user know which page to go back to.